lib/beaker-answers/pe_conf.rb

```diff
@@ -157,10 +182,40 @@ def the_host_with_role(role)
         Array(h['roles']).include?(role.to_s)
       end
 
-      if found_hosts.length == 0 or found_hosts.length > 1
+      if (found_hosts.length == 0 or found_hosts.length > 1) and (role != 'pe_postgres')
         raise ArgumentError, "There should be one host with #{role} defined, found #{found_hosts.length} matching hosts (#{found_hosts})"
       end
       found_hosts.first
     end
+
+    def postgres_cert_answers(pe_conf, meep_schema_version)
+      case meep_schema_version
+      when '1.0'
```

Comment:
Since there is no difference in what is being set relative to meep schema, there's no need to duplicate it.

For now, when '1.0','2.0' should work for both these methods.

Reply:
I'm attempting to make it easier to maintain if we ever change those settings in the future for 2.0. If you think that probably wont happen anytime soon I can simplify this.

Reply:
For now I'd remove the duplication; leave the case and it's simple enough to change in the future if needed.